src/lib/Components/ArtistListItem.tsx

```diff
@@ -152,29 +120,31 @@ export class ArtistListItem extends React.Component<Props, State> {
   render() {
     const { isFollowedChanging } = this.state
     const { artist } = this.props
-    const { is_followed, initials, image, href } = artist
+    const { is_followed, initials, image, href, name, nationality, birthday, deathday } = artist
     const { url } = image
 
     return (
       <Theme>
-        <TouchableWithoutFeedback onPress={() => this.handleTap(href)}>
-          <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
-            <Avatar size="xs" src={url} initials={initials} />
-            <Spacer m={1} />
-            <TextContainer flexDirection="column" alignItems="flex-start">
-              {this.renderText()}
-            </TextContainer>
-            <Spacer m={1} />
-            <Button
-              variant={is_followed ? "secondaryOutline" : "primaryBlack"}
-              onPress={this.handleFollowArtist}
-              size="small"
-              loading={isFollowedChanging}
-            >
-              {is_followed ? "Following" : "Follow"}
-            </Button>
-          </Flex>
-        </TouchableWithoutFeedback>
+        <Box width="100%" py={1}>
```

Comment:
We don't want to add margin/padding inside of a component which extends outside of the component's limits. I.e. this `Box` with padding should wrap the `ArtistListItem` outside of the component not live inside the component. Therefore we can control the margin/padding in different use cases/styles and keeps the padding and margin consistent without duplicating it in some places.

Reply:
Other than this this looks great!